proto/regen/ecocredit/v1/state.proto

```diff
@@ -136,6 +136,11 @@ message BatchInfo {
 
   // issuance_date is the timestamp when the credit batch was issued.
   google.protobuf.Timestamp issuance_date = 7;
+
+  // enable_future_minting tells if it's possible to mint new credits in
+  // the future. Once the enable_future_minting is set to false, it can't be
+  // toggled any more.
+  bool enable_future_minting = 8;
 }
```

Comment:
if we have `enable_future_minting`, `max_supply` is not needed any more. so I removed `max_supply`.

Reply:
I actually think the default should be sealed. So if we want a different name let's use `open`

Reply:
It's in the state, so we control the defaults. Good naming is a challenge.
If we want to rename this, then let's use `open` everywhere.